dp/cloud/python/magma/radio_controller/tests/test_utils/active_mode_cbsd_builder.py

```diff
@@ -115,12 +122,16 @@ def with_grant(
         self,
         grant_id: str, state: GrantState,
         hb_interval_sec: int, last_hb_ts: int,
+        low_frequency_hz: int = 3500,
```

Comment:
if its Hz, then maybe lets make those values reflect the actual Band 48, so `3.5` GHz is `3500000` in Hz